ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -971,9 +943,9 @@ func (pg *DEXOnboarding) validateBondStrength() bool {
 	return ok
 }
 
-func (pg *DEXOnboarding) semiBoldLabelGrey3(gtx C, text string) D {
-	lb := pg.Theme.Label(values.TextSize16, text)
-	lb.Color = pg.Theme.Color.GrayText3
+func semiBoldLabelGrey3(th *cryptomaterial.Theme, gtx C, text string) D {
+	lb := th.Label(values.TextSize16, text)
+	lb.Color = th.Color.GrayText3
 	lb.Font.Weight = font.SemiBold
 	return lb.Layout(gtx)
```

Comment:
```suggestion
return layout.Inset{Bottom: values.MarginPadding5}.Layout(gtx, lb.Layout)
```

Reply:
I'd prefer for callers to specify how much `Bottom` inset is required.

Reply:
Any specific reason why? Just to be sure, so some caller have more bottom inset than others?

Reply:
Yes, I'm following the Figma designs.